spec/fixtures/constants.rb

```diff
@@ -553,6 +553,10 @@
 clang: error: linker command failed with exit code 1 (use -v to see invocation)
 )
 
+SAMPLE_BITCODE_LD = %Q(
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.